projects/cm_mcu/Semaphore.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * Semaphore.c
+ *
+ *  Created on: September 06, 2022
+ *      Author: pkotamnives
+ *      declare and create semaphores utilized aross the cm_mcu project
+ */
+
+#include "FreeRTOS.h"
```

Comment:
as far as I can tell these headers are already included in the "Semaphore.h" file so no need to include them 2x

Reply:
Okay. Done.